lib/plugins/aws/package/compile/events/alb/lib/validate.test.js

```diff
@@ -217,4 +282,32 @@ describe('#validate()', () => {
       });
     });
   });
+
+  describe('#validatePriorities', () => {
+    it('should throw if multiple events use the same priority', () => {
+      const albEvents = [
+        { priority: 1, listenerId: 'aaa', functionName: 'foo' },
+        { priority: 1, listenerId: 'aaa', functionName: 'bar' },
+      ];
+      expect(() => awsCompileAlbEvents.validatePriorities(albEvents)).to.throw(Error);
```

Comment:
Could we add a check for the error message so that this test won't succeed if the function itself throws an unrelated error?